spec/dummy/db/migrate/20221224005546_create_events.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEvents < ActiveRecord::Migration[6.1]
```

Comment:
```suggestion
class CreateEvents < ActiveRecord::Migration[6.0]
```

Reply:
The Dummy app is a Rails 6.1 app, but we run tests (and migrations) on a Rails 6.0 app too. That's why we need to declare this migration as 6.0.